Header header
int32 marker_id
float32[3] rvec